layers/meta-balena-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging/balenaos_disableuefi.cfg

```diff
@@ -0,0 +1,2 @@
+CONFIG_CMD_BOOTEFI=
+CONFIG_EFI_LOADER=
```

Comment:
If not using u-boot-ti-staging, then these can be removed

Reply:
Removed in the latest version of the PR.

Reply:
These still seem to be here?

Reply:
Removed now... Please confirm and resolve.

Reply:
Not resolved